packages/core/src/node/messaging/ipc-bootstrap.ts

```diff
@@ -22,6 +24,10 @@ import { checkParentAlive, IPCEntryPoint } from './ipc-protocol';
 checkParentAlive();
 
 const entryPoint = IPCEntryPoint.getScriptFromEnv();
+const ipcServer = process.argv[process.argv.length - 1];
```

Comment:
I don't know if my concern is valid, but I'd prefer erasing information about that pipe from the spawned processes somehow:

```suggestion
const ipcServer = process.argv.pop();
```

Also `argv` might not be the best mechanism to transfer the IPC server path? I know this is what I used for the plugin host in a different change but it was only because we had total control over said `argv`.

Here it might be better to use environment variables:

```suggestion
const ipcServer = process.env.THEIA_IPC_SERVER;
delete process.env.THEIA_IPC_SERVER;
```